src/platform/packages/private/kbn-reporting/public/share/integrations/csv/csv_export_config.tsx

```diff
@@ -168,5 +169,23 @@ export const getShareMenuItems =
         contentType: 'text',
         generateAssetURIValue: () => absoluteUrl,
       },
+      renderTotalHitsSizeWarning: (totalHits: number = 0) => totalHits >= (csvConfig?.maxRows || 0),
+      totalHitsSizeWarning: (
+        <EuiCallOut
+          size="s"
+          color="warning"
+          title={i18n.translate('reporting.share.csv.reporting.totalHitsSizeWarning.title', {
+            defaultMessage: 'You have exceeded the recommended row limit.',
```

Comment:
<img width="379" height="203" alt="Screenshot 2025-11-25 at 10 35 33 AM" src="https://github.com/user-attachments/assets/5361bb6b-86b0-4994-b29f-016ac2f30012" />

Is there a way to make it more detailed? Like: your requested export includes 110,000 rows, which exceeds the max of 5000